source/slang/slang-emit-cpp.cpp

```diff
@@ -1669,7 +1677,7 @@ void CPPSourceEmitter::_emitWitnessTableDefinitions()
                 m_writer->emit(getName(witnessTableVal));
             }
             else if (entry->getSatisfyingVal() &&
-                     isPointerOfType(entry->getSatisfyingVal()->getDataType(), kIROp_RTTIType))
+                     entry->getSatisfyingVal()->getDataType()->op, kIROp_RTTIHandleType)
```

Comment:
It sure looks like there is a typo bug here, since the new code seems to be using "operator comma" in a way that would make this condition be true in a lot of cases where it shouldn't be.

Do you mean to be comparing `...->op == kIROp_RTTIHandleType` like you do in the code below?

Reply:
Good catch here. Thanks!